client/opt/option.go

```diff
@@ -54,6 +55,9 @@ const (
 	// It is stored as bool.
 	EnableRouterClient
 
+	// EnableRouterServiceHandler is the router service handle region requestions option.
+	EnableRouterServiceHandler
```

Comment:
It is not easy for a user to know the relationship between EnableRouterClient and EnableRouterServiceHandler

Reply:
removed